pom.xml

```diff
@@ -130,7 +130,7 @@
     <findbugs.jsr305.version>2.0.1</findbugs.jsr305.version>
     <geronimo.version>2.0.0</geronimo.version>
     <grok.version>0.1.0</grok.version>
-    <gson.version>2.2.4</gson.version>
+    <gson.version>2.6.2</gson.version>
```

Comment:
Is it compatible? One of the challenge is that gson is exposes via cdap-api. So if the gson version upgrade is not compatible (not just API, but if there is any behavioral changes), then it breaks the API contract.

Reply:
I looked up the release notes. Everything seems compatible except this one note:
```
Added support for java.util.Currency, AtomicLong, AtomicLongArray, AtomicInteger, AtomicIntegerArray, AtomicBoolean. This change is backward-incompatible because the earlier version of Gson used the default serialization which wasn't intuitive. We hope that these classes are not used enough to actually cause problems in the field.
```
Do you think this will cause an issue?

Reply:
This update is no longer needed since I'm using `BasicThrowable` to propagate errors.